PennMobile/Dining/SwiftUI/Views/Insights/PredictionsGraph/VariableStepLineGraphView+PredictionSlopePath.swift

```diff
@@ -27,16 +27,26 @@ extension VariableStepLineGraphView {
 
         func path(in rect: CGRect) -> Path {
             var path = Path()
-
             path.move(to: CGPoint(
                 x: lastDataPoint.x * rect.maxX,
                 y: rect.maxY - (rect.maxY * lastDataPoint.y)
             ))
-
-            path.addLine(to: CGPoint(
-                x: predictionZeroPoint.x * rect.maxX,
-                y: rect.maxY - (rect.maxY * predictionZeroPoint.y)
-            ))
+            if slope == 0 {
+                path.addLine(to: CGPoint(
```

Comment:
Why does this have to be its own case? What is the behavior if slope is 0 and we use the regular `x: rect.maxX, y: rect.maxY * (1 - lastDataPoint.y - slope * (1 - lastDataPoint.x))` case?